tests/integration/kubernetes/runtimeclass_workloads/pod-guest-pull-in-trusted-storage.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: large-image-pod
+spec:
+  runtimeClassName: RUNTIMECLASS
```

Comment:
This should be `kata` and it gets automatically updated with the correct runtimeclass during testing IIRC

Reply:
Fixed.

Reply:
It still says `runtimeClassName: $RUNTIMECLASS`?

Reply:
We need to replace it with the value of kata-${KATA_HYPERVISOR} using envsubst, similar to how the new_pod_config function does.

Reply:
So if you look inside the other `runtimeclass_workloads` test pod specs you will see we have `runtimeClassName : kata` set. I think it's because in the CI we alias the `kata` runtimeClass to match the platform we are testing.

Reply:
Fixed.